index.html

```diff
@@ -1,11 +1,757 @@
+            least a trillion trillionfold — from a submicroscopic speck of
+            primordial energy to the size of a grapefruit — in less than a
+            cosmic eye-blink.
+          </p>
+          <p>
+            Almost as quickly, this pulse would subside, relaxing into ordinary
+            particles and radiation. All of normal cosmic history was still
+            ahead, resulting in today’s observable universe, a patch of sky and
+            stars billions of light-years across. “It’s often said that there is
+            no such thing as a free lunch,” Dr. Guth likes to say, “but the
+            universe might be the ultimate free lunch.”
+          </p>
+          <p>
+            Make that free lunches. Most of the hundred or so models resulting
+            from Dr. Guth’s original vision suggest that inflation, once
+            started, is eternal. Even as our own universe settled down to a
+            comfortable homey expansion, the rest of the cosmos will continue
+            blowing up, spinning off other bubbles endlessly, a concept known as
+            the multiverse.
+          </p>
+          <p>
+            So the future of the cosmos is perhaps bright and fecund, but do not
+            bother asking about going any deeper into the past.
+          </p>
+          <p>
+            We might never know what happened before inflation, at the very
+            beginning, because inflation erases everything that came before it.
+            All the chaos and randomness of the primordial moment are swept
+            away, forever out of our view.
+          </p>
+          <p>
+            “If you trace your cosmic roots,” said Abraham Loeb, a
+            Harvard-Smithsonian astronomer who was not part of the team, “you
+            wind up at inflation.”
+          </p>
+          <span style="font-size: 0.6em; color: #888;"
+            >A version of this article appears in print on March 18, 2014,
+            Section A, Page 1 of the New York edition with the headline: Space
+            Ripples Reveal Big Bang’s Smoking Gun. Order Reprints | Today’s
+            Paper | Subscribe</span>
+          <div class="last-icons">
+            <span>
+              <button class="button-div-right">READ 615 COMMENTS</button>
+            </span>
+            <span class="comment-icons">
+              <img src="Images/share-arrow-icon.png" alt="share icon">
+              <img src="Images/mail-icon.png" alt="mail icon">
+              <img src="Images/twitter-icon.png" alt="twitter icon">
+              <img src="Images/facebook-icon.png" alt="facebook icon">
+            </span>
+          </div>
+        </div>
+        <section class="section2">
+          <div class="space-articles font">
+            <table>
+              <th>More in Space & Cosmos</th>
+              <tr>
+                <td class="article-img">
+                  <img src="Images/space-image1.jpg" alt="first space image">
```

Comment:
`REQUIRED`

Images and videos should be enclosed by a `<figure>` tag with `<figcaption>` that gives a description of the embedded image or video